src/chat/infra/repo/MongoChatRepository.ts

```
@@ -0,0 +1,188 @@
+
+    return {
+      id: messageDoc._id.toString(),
+      content: messageDoc.content || '',
+      createdAt: messageDoc.createdAt || new Date(),
+    };
+  }
+
+  private async mapToDomain(chatDoc: PopulatedChatModel | IChatModel): Promise<Chat> {
+    const botId = process.env.BOT_ID || '681c757539ec003942b3f97e';
+    const isPopulated = (doc: any): doc is PopulatedChatModel =>
+      doc.users && doc.users[0] && 'name' in doc.users[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)